hawc/apps/form_library/models.py

```diff
@@ -0,0 +1,28 @@
+import reversion
+from django.db import models
+
+from ..myuser.models import HAWCUser
+
+
+# TODO: is this a good name? i changed it from dynamic form because it makes the naming scheme weird
+# for the actual form class (DynamicFormForm)
+class CustomDataExtraction(models.Model):
+    name = models.CharField(max_length=128)
+    description = models.TextField(blank=True)
```

Comment:
I think we should make this required.  We'll want some context for what these forms are.